containers/azure-functions-pwsh-7/README.md

```diff
@@ -0,0 +1,61 @@
+# Azure Functions & PowerShell 7
```

Comment:
You'll need to change this to just "PowerShell"

Reply:
Done